cwl/src/main/scala/cwl/CommandLineTool.scala

```diff
@@ -120,18 +120,20 @@ case class CommandLineTool private(
 
   def asCwl = Coproduct[Cwl](this)
 
+}
+
+object CommandLineTool {
+
   /**
     * Sort according to position. If position does not exist, use 0 per spec:
     * http://www.commonwl.org/v1.0/CommandLineTool.html#CommandLineBinding
     */
-  def orderedInputs: Seq[CommandPart] =
+  protected[cwl] def orderedForCommandLine(inputs: Array[CommandInputParameter]):Seq[CommandInputParameter] = {
     inputs.
+      filter(_.inputBinding.isDefined).
```

Comment:
expanding on the scaladoc above, something like "and if there is no input binding ignore the input parameter"